### ISSUE TITLE
[36] Zero-Shot Anomaly Detection via Batch Normalization

### ISSUE THREAD

**Issue by sy00n** [sy00n/DL_paper_review#41]
### Abstract
- normal training data에 drift가 생기는 상황, 특히 new normal에 대한 training data가 없는 상황을 위해 zero-shot AD setting이 개발되어 오고 있음.
- off-the-shelf deep anomaly detectors(such as deep SVDD)를 adapt시켜서 inter-related training data distributions in combination with batch normalization을 통해 unseen AD에 대한 zero-shot generalization 성능을 확보함. 
- We propose Adaptive Centered Representations (ACR), a lightweight zero-shot AD method that combines two simple ideas: batch normalization and meta-training.
    - 가정: "normal" sample의 majority를 가정하면서 randomly-sampled batch에는 abnormalities보다 normalities가 더 많을 것이다.
    - Batch normalization을 수행함으로써 이 normal sample을 center로 가도록, abnormal sample은 바깥쪽으로 가도록 함.
    - 이런 scaling and centering 방식은 input에 대한 domain shift에 강건함
    - meta-training scheme도 제안함
    - 이런 ACR 방식의 장점은 theoretically grounded, simple, domain-independent, and compatible with various backbone mode 라고 주장함.
    -
